src/lib/query.ts

```diff
@@ -3,6 +3,8 @@ import { BaseQuery } from './base-query';
 import { BaseQueryParameters, QueryOperation, QueryOperator, TaxonomyQueryOperation } from './types';
 export class Query extends BaseQuery {
   private _contentTypeUid?: string;
+  private _subQueries: Query[] = [];
```

Comment:
@harshithad0703 This is not used anywhere. Please remove this.

Reply:
ok